systest/scripts/install_tlc.sh

```diff
@@ -36,14 +36,13 @@ apt-get -y install \
     python-protobuf \
     protobuf-compiler \
     gosu
-apt-get clean
```

Comment:
Yes.   I believe that the `apt-get clean` was a leftover when this series of commands was copied out of a Dockerfile.